latex/main.tex

```diff
@@ -29,30 +47,86 @@ \subsection{Who are we?}
 \subsection{Our product}
 The Pollution Detection System is a web-based application that provides information about the pollution on earth. Both light pollution and air pollution are being monitored using this system.
 \\
-\indent
+The system uses so called "sensorhubs". Sensorhubs are a combination of multiple sensors inside of a box. These sensorhubs can be placed in a specific area to measure the pollution. These measurements can be monitored using the PDS website. This can be done by using the website's main feature, the map.
 The system uses sensors that can be placed in a specific area to measure the pollution. These measurements can be monitored using the PDS website. This can be done by using the website's main feature, the map.
+\\
+The system uses sensorhubs that can be placed in a specific area to measure the pollution. These measurements can be monitored using the PDS website. This can be done by using the website's main feature, the map.
 
 \pagebreak
 
 \section{The website and how it works}
```

Comment:
In dit hoofdstuk ga je al uitleggen hoe het werk, dat is ook het doel van de handleiding, dus dat voelt erg dubbel.